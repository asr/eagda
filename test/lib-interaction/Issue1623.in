top_command (cmd_load currentFile [".","../../std-lib/src/"])
goal_command 0 cmd_auto ""
