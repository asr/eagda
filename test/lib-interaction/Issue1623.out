Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Relation.Binary.IsStrictTotalOrder .Relation.Binary.Core.Dummy._â‰¡_ (Relation.Binary.StrictTotalOrder._<_ strictTotalOrder) " nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-status-action "")
(agda2-info-action "*Auto*" "" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> 